spec/lib/main_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../../lib/log_parser'
```

Comment:
As you are testing the class LogParser from `log_parser` file, it seems that you should rename your spec file name to `log_parser_spec.rb`.

What do you think?